### PR TITLE
update platform_channel package name

### PR DESCRIPTION
**Pull request by szakarias**

### CODE REVIEW

File: examples/platform_channel/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.example.flutter"
+    package="com.example.platformChannel"
```
Comment (jakobr-google) [inline examples/platform_channel/android/app/src/main/AndroidManifest.xml]: Nit: I don't know if it's enforced, but package names are usually in all lower-case (https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html) - so either com.example.platformchannel or com.example.platform_channel.

**Review by jakobr-google — APPROVED** [review]
LGTM

File: examples/platform_channel/android/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (szakarias) [inline examples/platform_channel/android/app/src/main/AndroidManifest.xml]: Me neither, let's go for com.example.platformchannel then.

**Review by jakobr-google — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**